css/properties.json

```diff
@@ -9375,7 +9375,149 @@
     "computed": "asSpecified",
     "order": "perGrammar",
     "status": "standard",
-    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/shape-rendering"  "stroke": {
+    "syntax": "",
+    "media": "visual",
+    "inherited": true,
+    "animationType": [
```

Comment:
It would be good to check if we should use this array or simply "by computed value", I'm not sure

Reply:
I think I’ll change it to “by computed value” because the actual behavior of `stroke` is nothing like the formal syntax.